build.gradle

```diff
@@ -1,3 +1,19 @@
+
+// Apply the java plugin to add support for Java
 apply plugin: 'java'
 apply plugin: 'eclipse'
 apply plugin: 'idea'
+
+// In this section you declare where to find the dependencies of your project
+repositories {
+    // Use 'maven central' for resolving your dependencies.
+    // You can declare any Maven/Ivy/file repository here.
+    mavenCentral()
+}
+
+
+// In this section you declare the dependencies for your production and test code
+dependencies {
+    compile 'com.google.guava:guava:17.0'
```

Comment:
yes makes sense, i have removed the dependent class in other proposal 
https://github.com/hr1383/playback-features/pull/1